<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandStrengthTester</class>
 <widget class="QMainWindow" name="HandStrengthTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HandStrengthTester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>1121</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/background.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player0_card1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player0_card2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player1_card2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player1_card1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player2_card2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player2_card1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player3_card2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player3_card1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player4_card2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player4_card1">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player5_card2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>160</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player5_card1">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>160</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player6_card2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>270</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player6_card1">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player7_card2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>390</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player7_card1">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>390</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="card1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="card4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="card3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="card2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="card5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>44</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/cardDeck.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_test_again">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>490</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Again</string>
    </property>
   </widget>
   <widget class="QLabel" name="player0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player7">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>390</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>270</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player5">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
